### DISCUSSION TITLE
Select and Execute Specific Chunks of SQL Queries Within Stored Procedures

### DISCUSSION THREAD

**Discussion by mskperera** [dbeaver/dbeaver#36028] [Q&A] [answered]
It would be great to have a feature for executing specific chunks of SQL queries within stored procedures. This would make debugging and testing more convenient by allowing selective execution of query blocks rather than running the entire procedure.

**Comment by ged-yuko** [comment, marked as answer]
Doesn't it already work exactly like your title says: select a fragment of script editor's text you want to execute as a unit and then use Run query command?

**Comment by E1izabeth** [comment]
@ged-yuko it does. The feature already exist
